// frontend/.angular/cache/18.2.8/frontend/vite/deps/@angular_material_select.js
import {
  MAT_SELECT_CONFIG,
  MAT_SELECT_SCROLL_STRATEGY,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER_FACTORY,
  MAT_SELECT_TRIGGER,
  MatSelect,
  MatSelectChange,
  MatSelectModule,
  MatSelectTrigger,
  matSelectAnimations
} from "./chunk-GZ4DLCF2.js";
import "./chunk-XPPTRYL6.js";
import "./chunk-4ZTCTBBT.js";
import "./chunk-5E5PWKDH.js";
import "./chunk-ZN67S7ZQ.js";
import {
  MatError,
  MatFormField,
  MatHint,
  MatLabel,
  MatPrefix,
  MatSuffix
} from "./chunk-G55Z3SOX.js";
import "./chunk-NTOPAOTM.js";
import {
  MatOptgroup,
  MatOption
} from "./chunk-2I2UQ5Z6.js";
import "./chunk-27F3AP6P.js";
import "./chunk-VLBMOUSB.js";
export {
  MAT_SELECT_CONFIG,
  MAT_SELECT_SCROLL_STRATEGY,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER_FACTORY,
  MAT_SELECT_TRIGGER,
  MatError,
  MatFormField,
  MatHint,
  MatLabel,
  MatOptgroup,
  MatOption,
  MatPrefix,
  MatSelect,
  MatSelectChange,
  MatSelectModule,
  MatSelectTrigger,
  MatSuffix,
  matSelectAnimations
};
