// frontend/.angular/cache/18.2.8/frontend/vite/deps_ssr/@angular_material_select.js
import { createRequire } from 'module';const require = createRequire(import.meta.url);
import {
  MAT_SELECT_CONFIG,
  MAT_SELECT_SCROLL_STRATEGY,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER_FACTORY,
  MAT_SELECT_TRIGGER,
  MatSelect,
  MatSelectChange,
  MatSelectModule,
  MatSelectTrigger,
  matSelectAnimations
} from "./chunk-SQIKB6D2.js";
import "./chunk-EMAP2XRB.js";
import "./chunk-A4EMWIWZ.js";
import "./chunk-CP5OP27P.js";
import "./chunk-QLFQYEQQ.js";
import {
  MatError,
  MatFormField,
  MatHint,
  MatLabel,
  MatPrefix,
  MatSuffix
} from "./chunk-3TDSIH4E.js";
import "./chunk-Q4PCYS43.js";
import {
  MatOptgroup,
  MatOption
} from "./chunk-SFF6SVWP.js";
import "./chunk-6K3IK2DV.js";
import "./chunk-TW4XVQZD.js";
import "./chunk-NQ4HTGF6.js";
export {
  MAT_SELECT_CONFIG,
  MAT_SELECT_SCROLL_STRATEGY,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER_FACTORY,
  MAT_SELECT_TRIGGER,
  MatError,
  MatFormField,
  MatHint,
  MatLabel,
  MatOptgroup,
  MatOption,
  MatPrefix,
  MatSelect,
  MatSelectChange,
  MatSelectModule,
  MatSelectTrigger,
  MatSuffix,
  matSelectAnimations
};
