// frontend/.angular/cache/18.2.8/frontend/vite/deps_ssr/@angular_material_select.js
import { createRequire } from 'module';const require = createRequire(import.meta.url);
import {
  MAT_SELECT_CONFIG,
  MAT_SELECT_SCROLL_STRATEGY,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER_FACTORY,
  MAT_SELECT_TRIGGER,
  MatSelect,
  MatSelectChange,
  MatSelectModule,
  MatSelectTrigger,
  matSelectAnimations
} from "./chunk-7IB4XK77.js";
import {
  MatError,
  MatFormField,
  MatHint,
  MatLabel,
  MatPrefix,
  MatSuffix
} from "./chunk-SZSQXLHL.js";
import "./chunk-7VZ3LEYZ.js";
import "./chunk-2IE7MSW6.js";
import "./chunk-3SVW5GWY.js";
import "./chunk-QLFQYEQQ.js";
import "./chunk-KRJOORMQ.js";
import {
  MatOptgroup,
  MatOption
} from "./chunk-SQXJ67R2.js";
import "./chunk-TL65NOFV.js";
import "./chunk-TW4XVQZD.js";
import "./chunk-NQ4HTGF6.js";
export {
  MAT_SELECT_CONFIG,
  MAT_SELECT_SCROLL_STRATEGY,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER_FACTORY,
  MAT_SELECT_TRIGGER,
  MatError,
  MatFormField,
  MatHint,
  MatLabel,
  MatOptgroup,
  MatOption,
  MatPrefix,
  MatSelect,
  MatSelectChange,
  MatSelectModule,
  MatSelectTrigger,
  MatSuffix,
  matSelectAnimations
};
