// frontend/.angular/cache/18.2.8/frontend/vite/deps/@angular_material_select.js
import {
  MAT_SELECT_CONFIG,
  MAT_SELECT_SCROLL_STRATEGY,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER_FACTORY,
  MAT_SELECT_TRIGGER,
  MatSelect,
  MatSelectChange,
  MatSelectModule,
  MatSelectTrigger,
  matSelectAnimations
} from "./chunk-HYNL2FP2.js";
import {
  MatError,
  MatFormField,
  MatHint,
  MatLabel,
  MatPrefix,
  MatSuffix
} from "./chunk-WV2FEHHA.js";
import "./chunk-YQ3H65YI.js";
import "./chunk-TXMVHJZ2.js";
import "./chunk-FPAAPGA6.js";
import "./chunk-2N5IQ37S.js";
import "./chunk-5E5PWKDH.js";
import {
  MatOptgroup,
  MatOption
} from "./chunk-7GYOQT6C.js";
import "./chunk-Y4SMY5TR.js";
import "./chunk-VLBMOUSB.js";
export {
  MAT_SELECT_CONFIG,
  MAT_SELECT_SCROLL_STRATEGY,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER,
  MAT_SELECT_SCROLL_STRATEGY_PROVIDER_FACTORY,
  MAT_SELECT_TRIGGER,
  MatError,
  MatFormField,
  MatHint,
  MatLabel,
  MatOptgroup,
  MatOption,
  MatPrefix,
  MatSelect,
  MatSelectChange,
  MatSelectModule,
  MatSelectTrigger,
  MatSuffix,
  matSelectAnimations
};
